addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -567,6 +567,7 @@ var SnippetEditor = Widget.extend({
         // FIXME should not this call _destroyEditor ?
         activateSnippetProm.then(() => this.destroy());
         $parent.trigger('content_changed');
+        this.options.wysiwyg.options.onWysiwygChange && this.options.wysiwyg.options.onWysiwygChange();
```

Comment:
We already trigger `content_changed` for conceptually the same reason: notifying other parts of the code that the content here as changed. I think we should not need two different calls for this. I see two options:
A. Hook ourselves on the already existing `content_changed` mechanism.
B. Trigger `content_changed` in `onWysiwygChange` but that seems more dangerous in the sense that I feel like we may easily break things if we want to re-define a custom `onWysiwygChange`, as we'll need to call the "default" one anyway to continue triggering `content_changed`.

Reply:
`content_changed` is called on every `observerApply`. This would commit changes on each keystrokes, which we want to avoid as it would be expensive for Owl. We should only change the content on blur or on events that changes the documents from the sidebar or some helper buttons (like the one that duplicate or move a snippet) as they do not trigger a blur event.